packages/dds/map/src/test/mocha/map.fuzz.spec.ts

```diff
@@ -114,7 +105,11 @@ function makeGenerator(optionsParam?: Partial<GeneratorOptions>): AsyncGenerator
 	const setKey: Generator<SetKey, State> = ({ random }) => ({
 		type: "setKey",
 		key: random.pick(keyNames),
-		value: random.bool() ? random.integer(1, 50) : random.string(random.integer(3, 7)),
+		value: random.pick([
+			(): number => random.integer(1, 50),
+			(): string => random.string(random.integer(3, 7)),
+			(): IFluidHandle | undefined => random.handle(),
```

Comment:
I don't know if I love this, but it seemed like the most straightforward way to still allow disabling handles.

Reply:
left a comment below, i don't think we need to block this. disabling is more about the check where we fail if none are used so that tests that are broken to not be forced to have them. if you really want to be explcity, then it should throw and fail the test

Reply:
makes sense, I thought we wanted to not create any if it's disabled but that's fine too